update-links.js

```diff
@@ -0,0 +1,76 @@
+const fs = require("fs/promises")
+const F = require("fluture")
+const path = require("path")
+const {
+  curry,
+  filter,
+  fork,
+  identity: I,
+  j2,
+  lines,
+  map,
+  pipe,
+  readFile,
+  split,
+  always: K,
+  chain,
+} = require("snang/script")
+
+// writeFile :: String -> String -> Future Error String
+const writeFile = curry(
+  (file, raw) =>
+    new F.Future((bad, good) => {
+      fs.writeFile(file, raw, "utf8").catch(bad).then(good)
+      return () => {}
+    })
+)
+
+const INPUT = "./linktree"
+const OUTPUT = "./src/pages/links.js"
+
+const writeRaw = pipe(writeFile(OUTPUT), map(K(`Updated ${OUTPUT}!`)))
+const trim = x => x.trim()
+
+module.exports = pipe(
+  readFile,
+  map(
+    pipe(
+      lines,
+      filter(I),
+      map(pipe(split(": "), map(trim))),
+      j2,
+      z => `/* DO NOT EDIT THIS FILE DIRECTLY, INSTEAD LOOK AT update-links.js */
```

Comment:
Is there a reason we want the output to be a file instead of just outputting the component directly in JS?